src/Tools/babylon.observable.ts

```diff
@@ -154,6 +154,27 @@
 
             return result;
         }
+
+        public hasPickMasks(): boolean {

```

Comment:
What do you mean by too specific ? They shouldn't exist ? They should be private ? 
I need them in order to optimize the pickUp callback in the initClickEvent function. This function checks all functionalities that it should for the current configuration and returns a ClickInfo which warn to observables and actionManager what must be raised. The functionalities checked depends on the type of the triggers registered in ActionManager (where the necessity to know what kind of triggers are registered without processing a mesh pick thanks to the static functions HasPickTriggers and HasSpecificTrigger) and in the same way, it depends on the type of observer masks registered in PointerObservable and PrePointerObservable. I don't want to check if a double click has happened if nothing requested it.